src/pages/cw-multi-test/getting-started/counter/counter-sylvia.mdx

```diff
@@ -0,0 +1,468 @@
+---
+tags: ["multitest", "getting started", "counter", "Sylvia"]
+---
+
+import { Callout, Card, Cards } from "nextra/components";
+
+# Counter written in Sylvia
+
+## Counter source code
+
+The following code snippets present the content of [**Cargo.toml**](#cargotoml),
+[**lib.rs**](#librs), [**msg.rs**](#msgrs), and [**contract.rs**](#contractrs) files, respectively.
+You can just copy and paste the provided content to previously created empty files, temporarily
+skipping the detailed explanations. However, if you're curious about what happens inside each file,
+feel free to check the detailed explanations provided for each code snippet.
+
+### `Cargo.toml`
+
+```toml showLineNumbers filename="Cargo.toml" copy
+[package]
+name = "counter"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+# use library feature to disable all instantiate/execute/query exports
+library = []
+
+[dependencies]
+cosmwasm-schema = "2"
+cosmwasm-std = { version = "2" }
+cw-storage-plus = "2"
+schemars = "0.8"
+serde = "1.0"
+sylvia = { version = "1" }
+
+[dev-dependencies]
+sylvia = { version = "1", features = ["mt", "cosmwasm_2_0"] }
```

Comment:
This applies also to the CosmWasm example, but do we need `cosmwasm_2_0` feature here?

Reply:
In this example, it is indeed not needed. But as I remember, all features set here will be also passed to MT right? I want to be sure because I remember it was working this way...

Reply:
yes, they will be, but still in MT it shouldn't be needed neither. correct me if I'm wrong

Reply:
In this example it is definetely not needed. Anyway, I expect that this will be required when some specific options from MT are used in tests, but now I have no Sylvia example for it. Let's remove it now as you suggest.